src/System Application/App/Edit in Excel/src/EditinExcelImpl.Codeunit.al

```diff
@@ -865,13 +867,26 @@ codeunit 1482 "Edit in Excel Impl."
         if not TenantWebService.Get(TenantWebService."Object Type"::Page, ServiceName) then
             exit;
 
-        EditinExcelFilters.ReadFromJsonFilters(Filter, Payload, TenantWebService."Object ID");
+        EditinExcelFilters.ReadFromJsonFilters(Filter, Payload, TenantWebService."Object ID", FilterErrors);
         EditinExcel.OnEditInExcelWithFilters(ServiceName, EditinExcelFilters, SearchString, Handled);
         if Handled then begin
             Session.LogMessage('0000IG8', EditInExcelHandledTxt, Verbosity::Normal, DataClassification::SystemMetadata, TelemetryScope::ExtensionPublisher, 'Category', EditInExcelTelemetryCategoryTxt);
             exit;
         end;
-
+        if FilterErrors.Count() > 0 then
+            Message(StrSubstNo(EditInExcelInvalidFilterErr, FormatFilterErrors(FilterErrors)));
```

Comment:
Minor: `Message` has an overload where you can specify the values to replace in the message without having to use `Strsubstno`. Functionality is the same, but it gains a bit in readability.

```suggestion
            Message(EditInExcelInvalidFilterErr, FormatFilterErrors(FilterErrors));
```